src/main/java/com/afornalik/ox/controller/PlayerController.java

```diff
@@ -0,0 +1,56 @@
+package com.afornalik.ox.controller;
+
+import com.afornalik.ox.model.board.FieldStatus;
+import com.afornalik.ox.model.player.PlayerContainer;
+import com.afornalik.ox.view.UIOperations;
+
+import java.util.HashMap;
+import java.util.Map;
+
+class PlayerController {
+
+    private final UIOperations uiOperations;
+
+    PlayerController(UIOperations uiOperations) {
+        this.uiOperations = uiOperations;
+    }
+
+    String createPlayer() {
+
+        Map<String,Object> playerAttributes = new HashMap<>();
+
+        uiOperations.print("Player name : ");
+        playerAttributes.put("name",uiOperations.read());
```

Comment:
Why using map here and than calling other method just to convert such map to object? Wouldn't it be easier to add values to the player object from the beginning?

Reply:
I was thinking about validating this input  in PlayerContainer class